packages/blade/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { CardFooter, CardFooterLeading, CardFooterTrailing } from './CardFooter';
+import {
+  CardHeader,
+  CardHeaderIcon,
+  CardHeaderCounter,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderBadge,
+  CardHeaderIconButton,
+  CardHeaderLink,
+  CardHeaderText,
+} from './CardHeader';
```

Comment:
No it won't cause any issues, bundlers are smart enough to deduce what components are actually being used and do the code elimination. 

Tho i'll move it to index.ts because there's no point on keeping it here anymore since we moved away from the compound components.